java/src/jmri/util/usb/UsbBrowserFrame.java

```diff
@@ -9,8 +10,12 @@
 public class UsbBrowserFrame extends JmriJFrame {
 
     public UsbBrowserFrame() {
+        this(new UsbBrowserPanel());
+    }
+
+    public UsbBrowserFrame(JPanel rootPanel) {
```

Comment:
I need to make this new constructor package protected before committing this.  The intention is not to change the public interface.